### ISSUE TITLE
bug: only ZU_DEFAULT_USERNAME=admin works

### ISSUE THREAD

**Issue by update-freak** [dec0dOS/zero-ui#206]
# Bug Report

**ZeroUI version:** latest (v.1.5.8)

**Current behavior:**

I can only use the ZU_DEFAULT_USERNAME=admin.
When I try to change the username to another name (this I did in the stack in portainer) it did not work.

**Expected behavior:**

Changed username should then be used

**Steps to reproduce:**

Use the stack I attached and then try to change username

**Other information:**
I installed it on a Synology NAS
[zerotier.txt](https://github.com/dec0dOS/zero-ui/files/14401597/zerotier.txt)

**Comment by aruznieto** [comment]
Hi, how are you writing the name? I installed zero-ui in my Synology NAS and It works perfectly with other username configured

**Comment by update-freak** [comment]
I wort it like - ZU_DEFAULT_USERNAME=Felix
I tried to change it with portainer with the attached docker compose from admin to Felix, but this did not work.

[docker-compose.txt](https://github.com/dec0dOS/zero-ui/files/14485756/docker-compose.txt)

**Comment by aruznieto** [comment]
Try with `felix` instead of `Felix`

**Comment by update-freak** [comment]
yes, this approach (name start with small letter) I already tried but did not solve the problem

**Comment by update-freak** [comment]
maybe it's a related to missing rights on my side.
Did you tried to change the name after you created a stack or  you initially set another name?

**Comment by aruznieto** [comment]
> maybe it's a related to missing rights on my side. Did you tried to change the name after you created a stack or you initially set another name?

No
